src/components/layout/Container.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import Header from './header'
+
+const LeaderBoard = (props) => {
+    return (
```

Comment:
If we are immediately returning from the component I think it is much better to use a simplified arrow expression

for e.g

```javascript
const Component = props => 
    <div></div>
```

rather than using `{}`. Check `LeaderBoardItem.js` for example

Reply:
done